tests/playwright/scripts/setup-insecure-registry.sh

```
@@ -0,0 +1,144 @@
+# /**********************************************************************
```

Shellcheck (0.11.0)

[error] This shebang was unrecognized. ShellCheck only supports sh/bash/dash/ksh/'busybox sh'. Add a 'shell' directive to specify.

(SC1008)